app/views/articles/index.html.erb

```diff
@@ -10,7 +9,8 @@ Our blog has <%= Article.public_count %> articles and counting!
   <% @articles.each do |article| %>
     <li>
       <%= link_to article.title, article %> Created by:
-      <%= User.find_by(id: article.user_id).username %>
+      <%= link_to User.find_by(id: article.user_id).username,
```

Comment:
@idprietop Quering the DB in views or helper method is not a good approach. Please correct or define a global current_user method